src/Sentry/Laravel/Features/LivewirePackageIntegration.php

```diff
@@ -32,11 +31,56 @@ public function isApplicable(): bool
     }
 
     public function onBoot(LivewireManager $livewireManager): void
+    {
+        if (class_exists(EventBus::class)) {
```

Comment:
There is no version number, so we check for the `EventBus` class introduced in Livewire v3.